### PR TITLE
Added Valiant Rescuer

### PR DESCRIPTION
**Pull request by Emigara**
please test this card as I still am having problems loading client in eclipse

### CODE REVIEW

File: Mage.Sets/src/mage/cards/v/ValiantRescuer.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	public ValiantRescuer(UUID ownerId, CardSetInfo setInfo) {
+		super(ownerId, setInfo, new CardType[] {CardType.CREATURE}, "{1}{W}");
+		
+		this.subtype.add(SubType.HUMAN);
+		this.subtype.add(SubType.SOLDIER);
+		
+		this.power = new MageInt(3);
+		this.toughness = new MageInt(1);
+		
+		this.addAbility(new CycleControllerTriggeredAbility(
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/v/ValiantRescuer.java:33]: This needs to trigger only the first time something cycles in a turn.

Comment (Emigara) [reply Mage.Sets/src/mage/cards/v/ValiantRescuer.java:33]: is there a similar card with a first time int he turn condition on it so i can look and learn how to implement it?

Comment (theelk801) [reply Mage.Sets/src/mage/cards/v/ValiantRescuer.java:33]: Take a look at cards like [these](https://scryfall.com/search?q=o%3A%22for+the+first+time%22), it's sort of complicated which is why I hadn't gotten to it yet.

Comment (Emigara) [reply Mage.Sets/src/mage/cards/v/ValiantRescuer.java:33]: got it from the surveil first time in a turn card.  godda make a watcher nd use that to check the condition

**Pull request closed without merging**